src/TestCentric/testcentric.gui/AppEntry.cs

```diff
@@ -95,6 +95,7 @@ public static int Main(string[] args)
             new TestsNotRunPresenter(view.TestsNotRunView, model);
             new TextOutputPresenter(view.TextOutputView, model);
 			new TreeViewPresenter(view.TreeView, model);
```

Comment:
Not changed in this PR, but for consistency
```suggestion
            new TreeViewPresenter(view.TreeView, model);
```